### ISSUE TITLE
Kernel reading and writing to AnnData

### ISSUE THREAD

**Issue by Marius1311** [theislab/cellrank#900]
...
I'm trying to find a convenient way to load kernels for our tutorials. I could write the entire kernel to file and potentially read it in again somehow, however, that would require us to store this file on figshare which seems a bit unnecessary. I would much prefer if I could write the transition matrix to AnnData at the end of a kernel tutorial, and then reconstruct a new kernel from it (with transition matrix) in a new tutorial. For this, I tried `vk.write_to_adata()` where `vk` is a `VelocityKernel` with computed transition matrix. It works, the transition matrix is written to the underlying AnnData object. However, then I create a new `VelocityKernel` from the same AnnData object, it does *not* contain a transition matrix: 
```python
# initialize a new kernel based on this object
vk_1 = cr.kernels.VelocityKernel(adata)
```
but `vk_1.transition_matrix` is empty. Is this the intended behavior? Is there a better way to do what I'm trying to achieve?

**Comment by Marius1311** [comment]
But that's a bit counterintuitive, no? Because to initialize that `VelocityKernel`, I need to supply it with an `AnnData` object first, no?

**Comment by michalk8** [comment]
closed via #916
